package.json

```diff
@@ -21,7 +21,7 @@
     "script:migrate": "forge script script/mainnet/migration/MigrateToNative.s.sol --slow --via-ir",
```

Comment:
<img width="311" alt="image" src="https://github.com/user-attachments/assets/5840789c-ab73-4f4e-ad2d-7493c6c67410">
Is it me or lint commit messages in CI is checking way more than that? Could we also change that?

Reply:
Not sure if I get what you mean. In case you're talking about the number of the commits being checked, it is only checking the last one:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/f17bdc97-2066-44d8-a98c-b6d0aaa28c3a">
It matches the last one:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/2aea1efa-46aa-4b89-b5fe-f672b5f09e97">

If it wasn't like that, the check would also fail like was happening before. Is it that what you are pointing out?

Reply:
Agreed, changed the workflow name to `Lint`